common/changes/@rushstack/heft/ianc-rename-tests-lib-property_2020-09-28-09-21.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "[BREAKING CHANGE] Rename \"emitFolderNameForJest\" to \"emitFolderNameForTests\" in typescript.json",
```

```suggestion
      "comment": "(BREAKING CHANGE) Rename \"emitFolderNameForJest\" to \"emitFolderNameForTests\" in typescript.json",
```

We used `(` `)` earlier in the change log